sci-libs/lis/lis-1.6.5.ebuild

```diff
@@ -19,18 +19,13 @@ DEPEND="${RDEPEND}"
 
 PATCHES=( "${FILESDIR}"/${PN}-1.6.2-autotools.patch )
 
+pkg_pretend() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+}
+
 pkg_setup() {
 	if [[ ${MERGE_TYPE} != binary ]] && use openmp; then
```

Comment:
I don't love it normally but let's just use the same pattern as elsewhere but && FORTRAN...